{{ cookiecutter.repo_name }}/docs/source/conf.py

```diff
@@ -43,7 +43,8 @@
     'matplotlib.sphinxext.plot_directive',
     'numpydoc',
```

Comment:
As mentioned in previous comments, it looks like this numpydoc and napoleon don't play nice together, so I am suggesting that numpydoc be removed here.

```suggestion
```